api/contributors.json

```diff
@@ -7,4 +7,12 @@
         "linkedin_link": "https://www.linkedin.com/in/vedika-agarwal-a134311a9",
         "contributor_photo": "https://avatars.githubusercontent.com/u/67699840?v=4"
     }
-]
\ No newline at end of file
+    {
```

Comment:
This time you should add a comma here and my bad those should definately be deleted

Reply:
Comma on the above line

Reply:
Is it fine now @ArunTeltia ?